applications/asset_tracker_v2/src/events/location_module_event.h

```diff
@@ -144,6 +135,24 @@ struct location_module_wifi_access_points {
 };
 #endif
 
+/** @brief Location module data for associated payload for event of
+ *         LOCATION_MODULE_EVT_CLOUD_LOCATION_DATA_READY type.
+ */
+struct location_module_cloud_location {
+	/** Neighbor cell information is valid. */
+	bool neighbor_cells_valid;
+	/** Neighbor cells. */
+	struct location_module_neighbor_cells neighbor_cells;
```

Comment:
We could have just removed `struct location_module_neighbor_cells` and included the content in this structure.
I don't know which way to go. The downside of the current approach is that the names to access these variables gets really long such as `evt->data.cloud_location.neighbor_cells.cell_data.current_cell.rsrq` but I guess this shouldn't decide our choice.
@simensrostad Please let me know your thoughts.

Reply:
Its a bit long I agree, but on the other hand its also nice to keep separate structures for readability. I don't have a strong preference here. You decide.